lib/ui/text/paragraph_builder.cc

```diff
@@ -438,8 +438,7 @@ void ParagraphBuilder::pushStyle(tonic::Int32List& encoded,
   }
 
   if (mask & tsFontFamilyMask) {
-    style.font_families.insert(style.font_families.end(), fontFamilies.begin(),
-                               fontFamilies.end());
+    style.font_families = fontFamilies;
```

Comment:
Can you add a note here stating that fallback to system is intended?

Reply:
done